app/src/main/res/values-cy/help.xml

```diff
@@ -0,0 +1,135 @@
+yn araf trwy amser, neu sawl tro i deithio\'n gynt ac yn gynt.&lt;br/&gt;
+
+Gellir defnyddio Teithio Amser i ateb ystod o gwestiynnau, gan gynnwys
+\'Pa bethau fydd yn weladwy jyst wedi i\'r haul machlud heno?\', \'Pa 
+bryd nesaf bydd cyfle da i weld y blaned Mercher?\', ac \'Yn ôl 
+Hesiod, pa bryd dylwn gynaeafu fy ngrawnwin?\'&lt;br/&gt;&lt;br/&gt;
+
+&lt;b&gt;Oriel Hubble&lt;/b&gt;&lt;br/&gt;
+Cynwysa Sky Map oriel o ddelweddau o delesgop Hubble. Gellir pori
+neu chwilio\'r delweddau yn yr Oriel.&lt;br/&gt;&lt;br/&gt;
+
+&lt;b&gt;Modd Golwg Nos&lt;/b&gt;&lt;br/&gt;
+Pwrpas modd golwg nos yw gwarchod dy lygaid wrth ddefnyddio\'r ap yn y 
+tywyllwch. Yn y modd hwn, mae Sky Map yn pylu\'r sgrin ac yn defnyddio coch
+ar gefndir ddu. Mae dewis yr opsiwn eto yn toglo nôl i\'r modd arferol.&lt;br/&gt;&lt;br/&gt;
+
+&lt;b&gt;Gosod Dy Leoliad&lt;/b&gt;&lt;br/&gt;
+Mae ar Sky Map angen gwybod dy leoliad bras er mwyn dangos map o\'r awyr 
+fel y mae i\'w weld o dy leoliad. Yn rhagosodedig, mae\'n adnabod dy leoliad 
+o dy rwydwaith symudol. Os nad oes signal neu os nad wyt wedi galluogi
+rhwydwaith, defnyddir GPS dy ddyfais. Gellir gosod dy leoliad dy hun yng 
+ngosodiadau\'r ap, naill ai wrth roi enw lle neu gyfesurynnau (lledred a 
+hydred). Noder bod ar Sky Map angen cysylltiad data neu wi-fi i gael dy 
+gyfesurynnau o enw lle.&lt;br/&gt;&lt;br/&gt;
+
+&lt;b&gt;Cyfyngiadau a Datrys Problemau&lt;/b&gt;&lt;br/&gt;
+&lt;i&gt;Dyw\'r map ddim yn gywir.&lt;/i&gt;&lt;br/&gt;
+Gall fod nifer o resymau pam nad yw\'r map yn dangos yr awyr yn gwbl gywir:&lt;br/&gt;
+1) Ydy dy gwmpawd wedi\'i raddnodi? Dalia dy ddyfais ac wrth droi dy arddwn symud y 
+ddyfais mewn patrwm rhif wyth i galibro ei synwyryddion cyfeiriad. Mae\'n bosib 
+bydd angen ail-adrodd hyn. Nodwedd o\'r ddyfais yw hyn, nid Sky Map.&lt;br/&gt;
+2) Ydy\'r ap yn gwybod dy leoliad cywir? Ydy GPS neu leoliad rhwydwaith dy ddyfais
+ymlaen?&lt;br/&gt;
+3) Ydy\'r amser a chylchfa amser cywir wedi\'i osod? Mae Sky Map yn cyfrifo amser 
+GMT ac mae angen gwybod yr amser a dy gylchfa amser i wneud hyn. Os oes cerdyn 
+SIM yn dy ddyfais, bydd rhain fel arfer wedi eu gosod yn gywir.&lt;br/&gt;
+4) Ydy cywiriad magnetig wedi\'i droi ymlaen? Mewn rhai rhannau o\'r byd gall 
+y gwahaniaeth rhwng gogledd cywir a gogledd magnetig fod cymaint a 20 gradd.&lt;br/&gt;
+&lt;i&gt;Dyw\'r map ddim yn symud.&lt;/i&gt;&lt;br/&gt;
+Wyt ti ym modd llywio â llaw? Os ddim, gall hyn fod oherwydd cwmpawd heb ei galibro.
+Dilyna\'r camau uchod i gael dy ddyfais i adnabod y cyfeiriad yn gywir.&lt;br/&gt;&lt;br/&gt;
+
+&lt;b&gt;Clodrestr&lt;/b&gt;&lt;br/&gt;
+&lt;i&gt;Peirianneg, Dylunio a Rheolaeth Cynnyrch&lt;/i&gt;&lt;br/&gt;
+Brent Bryan, Dominic Widdows, Hector Ouilhet, James Powell,
+John Taylor, Kevin Serafini, Lior Ron&lt;br/&gt;&lt;br/&gt;
+
+&lt;i&gt;Cyfranwyr Cod Agored&lt;/i&gt;&lt;br/&gt;
+Christopher Wilcox, PhD&lt;br/&gt;&lt;br/&gt;
+
+&lt;i&gt;Lleoleiddiad Cymraeg&lt;/i&gt;&lt;br/&gt;
```

Comment:
Glad to see you put this in @Cymrodor 
I'm going to try to remember to suggest this for all translators.